docs/features/dead_letter_archive.md

```diff
@@ -20,6 +20,25 @@ Sending a POST request to this endpoint will trigger a Cumulus AsyncOperation th
 
 This endpoint may prove particularly useful when recovering from extended or unexpected database outage, where messages failed to process due to external outage and there is no essential malformation of each Cumulus message.
 
+### Configurable request parameters
+
+The DLA recovery endpoint takes the following configurations that help bound performance.
```

Comment:
I think we also need to update cumulus-api endpoint document.

Reply:
@jennyhliu  agree - I'll make those updates once this PR is approved.